solr/core/src/java/org/apache/solr/handler/component/QueryComponent.java

```diff
@@ -147,15 +147,6 @@ public void prepare(ResponseBuilder rb) throws IOException {
       }
     }
 
-    // Set field flags
```

Comment:
BTW, I cannot figure out why "field" is  used to describe/qualify the flags.  If you can't either, I recommend as you move this comment to simply omit the "field" word.

Reply:
Removed the comment in [5d842c6](https://github.com/apache/solr/pull/2221/commits/5d842c6bb7683346a5d1b575a9134e72702ddc10)